android/java/res/drawable/bat_funds_selector.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle"  >
+            <corners android:radius="24dip" />
+            <solid android:color="@color/brave_rewards_theme_color"/>
+        </shape>
+    </item>
+</selector>
```

Comment:
nit: new line and also add it at the end of each file in this PR

Reply:
updated